src/birdnetlib/analyzer.py

```diff
@@ -200,10 +206,15 @@ def analyze_recording(self, recording):
         start = 0
         end = recording.sample_secs
         results = {}
+        features = []
         for c in recording.chunks:
 
             if self.use_custom_classifier:
                 pred = self.predict_with_custom_classifier(c)[0]
+            elif self.fetch_embeddings == True:
```

Comment:
@MJWeldy From looking at this it appears that custom classifier functionality and the embeddings functionality are not possible at the same time. Is that correct?

I haven't used embeddings myself, so not sure if that is even relevant.

Reply:
Thanks. I'll try to get to adding a test case in the next few weeks. 

The licensing is fine with me. At this point I need to get my current project (which uses the older 320 length embeddings) over the line before I switch to the lib library. The way I added the feature extraction does not allow for the custom classifier and embedding extraction in the same run. But it would be fairly trivial to allow it to return both. I can think of a few niche case where someone might want both (like someone trying to visualize the embeddings for a few feature classes) while using a custom classifier head), but I think it would be pretty rare. 

I did not add the embeddings to the older tf.lite model. Also, it would be nice to adapt the multiprocessing functions to allow for embeddings. My current scripts embed 1000 audio files in  ~5-6 seconds.